src/theme/components/wishlist/WishlistProductToggleButton.vue

```diff
@@ -0,0 +1,52 @@
+<template>
+    <button class="btn btn-primary btn-circle absolute bottom-2 right-2" @click.prevent="toggleWishlist">
+        <client-only>
+            <ArrowPathIcon v-if="loading" class="h-5 w-5 animate-spin" />
+            <HeartIcon v-else class="h-5 w-5" :class="{'fill-green-100': isOnWishlist }" />
```

Comment:
use more themable class e.g. fill-accent

Reply:
i used fill-current, because fill-accent looks weird with the current background of the button